requirements/edx/github.txt

```diff
@@ -31,6 +31,7 @@ git+https://github.com/pmitros/pyfs.git@96e1922348bfe6d99201b9512a9ed946c87b7e0b
 git+https://github.com/hmarr/django-debug-toolbar-mongo.git@b0686a76f1ce3532088c4aee6e76b9abe61cc808
 # custom opaque-key implementations for ccx
 -e git+https://github.com/jazkarta/ccx-keys.git@e6b03704b1bb97c1d2f31301ecb4e3a687c536ea#egg=ccx-keys
+-e git+https://github.com/solashirai/crowdsourcehinter.git@b92d9fae317550b6942f201a9338e2bee3720064#egg=crowdsourcehinter-xblock
```

Comment:
I think that I added a tag to my commit then pushed... should there be any change I can see within github? Also is there some kind of tagging convention that I should follow for this?